astropy/table/table.py

```diff
@@ -2527,8 +2527,25 @@ def read(cls, *args, **kwargs):
           >>> dat = Table.read('table.dat', format='ascii')
           >>> events = Table.read('events.fits', format='fits')
 
-        The arguments and keywords (other than ``format``) provided to this function are
-        passed through to the underlying data reader (e.g. `~astropy.io.ascii.read`).
+        See http://docs.astropy.org/en/stable/io/unified.html for details.
+
+        Parameters
+        ----------
+        format : str
+            File format specifier.
+        *args : tuple, optional
+            Positional arguments passed through to data reader. If supplied the
+            first argument is the input filename.
+        **kwargs : dict, optional
+            Keyword arguments passed through to data reader.
+
+        Returns
+        -------
+        out : `Table`
+            Table corresponding to file contents
+
+        Notes
```

Comment:
Why the notes section without actual notes? Was this meant to have something added for both `read` and `write` after the class is defined?

Reply:
Yes, I got completely confused by this.  Turns out it was some past "cleverness" [my fault!] and these docstrings get munged to add a table of available formats.  So without the "Notes" section the content gets dumped into the `Return` or `Parameters` section and the sphinx output is broken.  And if you do a `Table.read?` you'll see the table of formats.

Reply:
OK, that is good to know -- but to ensure confusion doesn't recur, let's add a comment!